biblib/views/user_view.py

```diff
@@ -84,6 +84,7 @@ def get_libraries(cls, service_uid, absolute_uid, start=0, rows=None, sort_col="
                 .filter(Permissions.user_id == service_uid)\
                 .order_by(getattr(getattr(Library, sort_col), sort_order)())\
                 .all()
+            libraries_response = {'libraries_count': len(result)}
```

Comment:
Moving this because there was another `result` variable further down.

Reply:
That makes sense. It was a good idea to rename the result below to `owner_permissions`, `owner`. I don't think you need to break the tuple here, but it might be worth giving this a more descriptive name as well.

Reply:
Done.